backend/src/main/kotlin/cz/loono/backend/api/controller/HealthcareProvidersController.kt

```diff
@@ -0,0 +1,33 @@
+package cz.loono.backend.api.controller
+
+import cz.loono.backend.api.service.HealthcareProvidersService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class HealthcareProvidersController {
+
+    @Autowired
+    private lateinit var healthCareProvidersService: HealthcareProvidersService
+
+    @GetMapping(value = ["$DOCTORS_PATH/update"])
+    fun updateData() {
+        healthCareProvidersService.updateData()
+    }
+
+    @GetMapping(value = ["$DOCTORS_PATH/all"])
+    fun getAll() {
+        // TODO returns all providers in simple form
+    }
+
+    @PostMapping(value = ["$DOCTORS_PATH/detail"])
```

Comment:
`GetMapping`

Reply:
I suppose that there will be some parameters to get the concrete detail. So that's why I has written it as POST.